apps/pipeline/lib/pipeline/writer/table_writer/statement/create.ex

```diff
@@ -74,4 +105,8 @@ defmodule Pipeline.Writer.TableWriter.Statement.Create do
       value -> value
     end
   end
+
+  defp safe_col_name(col_name) do
+    String.replace(col_name, "-", "_")
```

Comment:
Andi validation should theoretically let the user know what is valid. Further detail could be added to the Andi Curator wiki docs.

I don't think this case warrants either validation or docs though because we case it for them.

I support checking for unique names with _ and - being equal could be a good validation card to add